blog/2020-10-08-FacilityLocationProblemQuickstart.adoc

```diff
@@ -0,0 +1,98 @@
+= New quickstart for the facility location problem
+:imagesdir: 2020/10/08
+:page-interpolate: true
+:awestruct-author: yurloc
+:awestruct-layout: blogPostBase
+:awestruct-tags: [use case, facility location problem, quarkus]
+
+:awestruct-share_image_filename: CreditDisputeProcessDiagram.png
+
+We have created a new https://github.com/kiegroup/optaplanner-quickstarts/tree/development/quarkus-facility-location[quickstart] that extends OptaPlanner use case portfolio with the https://en.wikipedia.org/wiki/Facility_location_problem[facility location problem] and also demonstrates building a cloud-native application using https://quarkus.io/[Quarkus], the new Java Native stack.
+
+== The facility location problem
+
+The facility location problem is about selecting a subset of potential facility locations so that they can provide services to a given set of consumers.
```

Comment:
"is about" is too uncertain. "selecting" is passive voice. How about something like:
"In the facility location problem, given a set of potential facility locations and customer locations, we need to decide which facility locations to develop to serve those customers."